community/Jim/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="Description" content="Enter your description here" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.6.0/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
+    <link rel="stylesheet" href="assets/css/style.css">
+    <title>Jim Morel</title>
+</head>
+
+<body>
+    <div class="jumbotron jumbotron-fluid">
+        <div class="container">
+            <div class="warning">
+                <img src="assets/images/lynx.png" alt="lynx">
+                <h1 class="display-4">Remember...</h1>
+            </div>
+            <p class="lead"><i>(This is an important warning message.)</i></p>
+        </div>
+    </div>
+    <h1></h1>
+    <div class="container">
+        <img style="width:75%;"" src="assets/images/land-rover.png" alt="its not a jeep">
```

Comment:
Hi @JimLynx small note here, we have a stray extra double-quote in the `img` tag. Can you resolve, and then we are good to merge. 🤜